lte/gateway/python/magma/pipelined/app/policy_mixin.py

```diff
@@ -182,13 +194,14 @@ def _process_redirection_rules(self, requests):
                 if rule.redirect.support == rule.redirect.ENABLED:
                     self._install_redirect_flow(imsi, ip_addr, rule)
 
-    def activate_rules(self, imsi, ip_addr, apn_ambr, static_rule_ids, dynamic_rules):
+    def activate_rules(self, imsi, msisdn: bytes, ip_addr, apn_ambr, static_rule_ids, dynamic_rules):
```

Comment:
check that test_enforcement_stats.py uses activate_rules directly so change the usage there as well